README.md

```diff
@@ -133,6 +133,60 @@ Migrating shouldn't be that difficult though. `UUIDTools::UUID` implements
 by default. But it's good to be aware of this in case you're running into
 weirdness.
 
+## Invalid Values
+
+By default, this will raise an exception if an invalid value is attempted to be stored.
+However, the PostgreSQL driver nils invalid UUID values.
+To get PG-friendly behavior, pass in the option `:nil_invalid_values => true` when creating the attribute.
+
+Additionally, as a convenience function, you can add the following to `ApplicationRecord` to get parity between PG and MySQL:
+
+```
+def self.uuid_fields(*fields)
+	if ActiveRecord::Base.connection.adapter_name.downcase.starts_with?("mysql")
+		fields.each do |fld|
+			attribute fld, MySQLBinUUID::Type.new(:nil_invalid_values => true)
+		end
+	end
+end
+```
+
+Then, in your models, just begin them by marking the fields which are UUIDs.
```

Comment:
Could you move this to as a section under `# Usage` (after `## Tell your model how to handle..`) please? I think we could just describe the options there, but don't need to specifically mention parity between Postgres and MySQL:

```
## Options
You can pass in the following options to `MySQLBinUUID::Type.new(option: "value")` when
registering the attribute:

  * `nil_invalid_values: true` (default: `false`) - When set to true, invalid UUIDs will be stored
    as `nil` values. When set the false (the default) an `InvalidUUID` error is raised. If you
    don't want `nil` values to be stored you can use a presence validator.
```